sovtoken/sovtoken/request_handlers/read_req_handler/get_utxo_handler.py

```diff
@@ -69,6 +70,10 @@ def get_result(self, request: Request):
 
         utxos = outputs.sorted_list
         next_seqno = None
+        if from_seqno:
+            idx = next((idx for utxo, idx in zip(utxos, range(len(utxos))) if utxo.seqNo == from_seqno), None)
+            if idx:
+                utxos = utxos[idx:]
         if len(utxos) > UTXO_LIMIT:
             next_seqno = utxos[UTXO_LIMIT].seqNo
             utxos = utxos[:UTXO_LIMIT]
```

Comment:
Please don't forget to use `MSG_LEN_LIMIT` from config file (either in this PR or another)

Reply:
Fixed this